source/_integrations/rflink.markdown

```diff
@@ -4,6 +4,9 @@ description: Instructions on how to integrate RFLink gateway into Home Assistant
 logo: rflink.png
 ha_category:
   - Hub
+ha_iot_class:
+  - Assumed State
+  - Local Push
```

Comment:
This cannot be multiple.

Reply:
I see.
Ok so I guess it is more appropriate to just leave the 'Assumed State' class.